browser-test/src/civiform_admin_publish.test.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  dismissModal,
+  startSession,
+  loginAsAdmin,
+  validateScreenshot,
+  AdminPrograms,
+  AdminQuestions,
+} from './support'
+import {Page} from 'playwright'
+
+describe('publishing all draft questions and programs', () => {
+  let pageObject: Page
+  let adminPrograms: AdminPrograms
+  let adminQuestions: AdminQuestions
+
+  const programNoQuestions = 'publish-test-program-no-questions'
+  const programWithQuestion = 'publish-test-program-with-question'
+  const questionName = 'publish-test-address-q'
+
+  beforeAll(async () => {
+    const {page} = await startSession()
+    pageObject = page
```

Comment:
question (blocking): why reassign?

Reply:
startSession returns an object containing multiple values. Presently, the variable we use to track the page (similar to other tests) is called pageObject. We need to store the result off in a single variable, then assign it to the variable.

Reply:
could you rename the variable defined in this file to `page` and edit L21 to be `{page} = await startSession()`?

Reply:
Unfortunately no. That turns out to be a compiler error:
```
src/civiform_admin_publish.test.ts:21:12 - error TS2809: Declaration or statement expected. This '=' follows a block of statements, so if you intended to write a destructuring assignment, you might need to wrap the the whole assignment in parentheses.
```

Switching the statement to:

```
({page} = await startSession()
```

works. That said, that pattern is not obvious to me and I think it might be better to be consistent with what we do elsewhere:

```
const session = await startSession()
pageObject = session.page